bazel/repository_locations.bzl

```diff
@@ -69,9 +76,9 @@ REPOSITORY_LOCATIONS = dict(
         urls = ["https://github.com/bazelbuild/bazel-skylib/archive/1.0.2.tar.gz"],
     ),
     io_bazel_rules_docker = dict(
-        sha256 = "1f4e59843b61981a96835dc4ac377ad4da9f8c334ebe5e0bb3f58f80c09735f4",
-        strip_prefix = "rules_docker-0.19.0",
-        urls = ["https://github.com/bazelbuild/rules_docker/archive/v0.19.0.tar.gz"],
+        sha256 = "4349f2b0b45c860dd2ffe18802e9f79183806af93ce5921fb12cbd6c07ab69a8",
+        strip_prefix = "rules_docker-0.21.0",
+        urls = ["https://github.com/bazelbuild/rules_docker/archive/v0.21.0.tar.gz"],
```

Comment:
This is required because there was a bug in the underlying `scala_rules` (https://github.com/bazelbuild/rules_docker/pull/1954)

Reply:
Because this has the potential to break things, let me try to pull this in as a separate change.

Reply:
82386b8 pulled this in.

Reply:
Thanks for doing that. Rebased this branch and removed this as an outstanding item to test.